src/node/node_string.hpp

```diff
@@ -48,9 +48,9 @@ inline String<node::Types>::String(const v8::Local<v8::String> &s) {
     if (s.IsEmpty() || s->Length() == 0) {
         return;
     }
-    m_str.resize(s->Utf8Length());
```

Comment:
The `Utf8Length` and `WriteUtf8` APIs now take an `v8::Isolate` - should we `ifdef` the calls or do we not care about transforming the strings as optimally as possible here?

Reply:
We don't have any benchmark which can guide us here. I suggest that we leave it as it is.